lib/key-generators/git-commit.js

```diff
@@ -0,0 +1,23 @@
+var CoreObject  = require('core-object');
+var gitRepoInfo = require('git-repo-info');
+var Promise     = require('ember-cli/lib/ext/promise');
+
+module.exports = CoreObject.extend({
+  generate: function() {
+    var path = gitRepoInfo._findRepo();
```

Comment:
It should be, yes.

`findRepo` starts looking from on the current working directory, and if that's not a git repository, it starts searching the parent directories. Presuming that you deploy from the root directory of your project's repo, it will always find the repository straight away.

The tests mirror this by `chdir`ring into the git repository fixture directories.